shared/fs/common/open-hoc.tsx

```diff
@@ -46,6 +46,7 @@ function OpenHOC(Component: React.ComponentType<Props & InjectedProps>): React.C
 
     return <Component {...props} onOpen={destinationPickerGoTo} />
   }
+  return Inner
```

Comment:
Does this make a difference?

Reply:
helps with react devtools, else you get like `_c()` as the name